packages/core/src/lib/Console.ts

```diff
@@ -306,10 +313,20 @@ class MemLabConsole {
       return;
     }
     if (this.config.isContinuousTest || !this.config.muteConsole) {
-      console.log(msg);
+      if (this.isTextOutput) {
+        console.log(msg);
+      }
+      else {
+        this.outStream.write(msg);
+        this.outStream.write('\n');
+      }
     }
   }
 
+  public writeRaw(s: string): void {
+    process.stdout.write(s);
+  }
```

Comment:
Modules other than `Console` should not directly write to the raw stream. Instead you can use `info.topLevel` which adds no text styles. The existing `Console` method like `info.topLevel` will instrument and log all messages to a log file besides writing to stdout.

Reply:
See my comment below